templates/todo/api/python/todo/models.py

```diff
@@ -20,11 +20,12 @@ def __init__(self, *args, **kwargs):
             credential = DefaultAzureCredential()
             keyvault_client = SecretClient(self.AZURE_KEY_VAULT_ENDPOINT, credential)
             for secret in keyvault_client.list_properties_of_secrets():
-                setattr(
-                    self,
-                    keyvault_name_as_attr(secret.name),
-                    keyvault_client.get_secret(secret.name).value,
-                )
+                if secret.name == "AZURE-COSMOS-CONNECTION-STRING":
```

Comment:
Because when using `infra/core/database` to create `Cosmos Mongo`, only one secret named "AZURE-COSMOS-CONNECTION-STRING" will be created in keyVault, the specific code is [here](https://github.com/Menghua1/azure-dev/blob/todo-nodejs-mongo-aks/templates/common/infra/bicep/core/database/cosmos/cosmos-account.bicep#L34). 
But now when using `AVM` to create `Cosmos Mongo`, multiple secrets are created and the specific code is [here](https://github.com/Azure/bicep-registry-modules/blob/main/avm/res/document-db/database-account/main.bicep#L497-L530). The backend will fail to obtain "AZURE-COSMOS-CONNECTION-STRING" in the secret. It will succeed only when `secret.name == "AZURE-COSMOS-CONNECTION-STRING"` is specified.